Sources/App/Views/DocumentationPageProcessor.swift

```diff
@@ -23,7 +23,8 @@ struct DocumentationPageProcessor {
     let repositoryName: String
     let packageName: String
     let reference: String
-    let referenceKind: Version.Kind?
+    let referenceLatest: Version.Kind?
+    let referenceKind: Version.Kind
```

Comment:
It might make sense to make `reference` of type `Reference` here and drop `referenceKind` because it can be accessed through `reference.versionKind` like above.

Reply:
Actually, that's probably not practical since `reference` here is from the input parameter `req.parameters.get("reference")`. While we _could_ construct a `Reference` from that it's probably not a good idea to do that here.